Source/WebInspectorUI/UserInterface/Views/FontDetailsPanel.js

```diff
@@ -220,6 +222,13 @@ WI.FontDetailsPanel = class FontDetailsPanel extends WI.StyleDetailsPanel
 
         let fontVariationPropertiesSection = new WI.DetailsSection("font-variation-properties", WI.UIString("Variation Properties", "Variation Properties @ Font Details Sidebar Section", "Section title for font variation properties."), [this._fontVariationsGroup]);
         this.element.appendChild(fontVariationPropertiesSection.element);
+
+        // Font Properties
+        let allFontsRow = new WI.DetailsSectionSimpleRow(WI.UIString("All Fonts", "All Fonts @ Font Details Sidebar Property", "Property title for all fonts used."));
+
+        this._allFontsGroup = new WI.DetailsSectionGroup([allFontsRow]);
+        let allFontsSection = new WI.DetailsSection("font-all-fonts-used", "All Fonts", [this._allFontsGroup]);
```

Comment:
we don't usually do piecemeal changes like this, where one PR adds some structure only for another PR to fill it in.  usually each PR is a self contained feature (or at least enough of one to be considered something substantive/independent).

i am very in favor of showing additional font information, but as-is this PR is not really complete enough to be approved.  once you have something to put inside this element then we can revisit the overall idea :)

Reply:
Okay thank you for clarifying, converted to a draft.